pageserver/src/rocksdb_storage.rs

```diff
@@ -114,11 +131,9 @@ impl ObjectStore for RocksObjectStore {
 impl RocksObjectStore {
     /// Open a RocksDB database.
     pub fn open(conf: &'static PageServerConf) -> Result<RocksObjectStore> {
-        let path = conf.workdir.join("rocksdb-storage");
-        let db = rocksdb::DB::open(&Self::get_rocksdb_opts(), path)?;
-
-        let storage = RocksObjectStore { _conf: conf, db };
-        Ok(storage)
+        let opts = Self::get_rocksdb_opts();
```

Comment:
This is an unrelated change, let's do it as a separate PR to slim down this one.

Reply:
Yes, this is part of GC changes. Will undo them.